styles/style.css

```diff
@@ -0,0 +1,182 @@
+body {
+    box-sizing: border-box;
+    background-color: #DCDCDC;
+}
+.header {
+    background-image: url("../images/header-image.jpg");
+    background-color: #003C63;
+    background-size: 100%;
+    height: 15rem;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+.caption {
+    display: block;
+    color: #42aa04;
+    font-style: italic;
+    line-height: 10%;
+}
+.checkout {
+    color: #FFFFFF;
+    background-color: #42aa04 ;
+}
+.header-box-1 {
+    text-align: center;
+    margin: 2rem 2rem 6rem 4rem;
+}
+.header-box-2  {
+    display: block; 
+    margin: 2rem 6rem 10rem 20rem;
+    align-items: center;
+}
+.header-box-2 i {
+    color: #FFFFFF;
+    flex-direction: row-reverse;
+    line-height: 1rem;
+}
+.adopt {
+    background-color: #42aa04;
+    color: #FFFFFF;
+    height: 3rem;
+}
+.main-menu {
+    display: flex;
+    flex-direction: row;
+    margin-left: 3rem;
+    margin-right: 30rem;
+    font-family: sans-serif;
+}
+li a {
+    text-decoration: none;
+    color: #2e3236;
+    font-size: 80%;
+    font-weight: bold;
+}
+li {
+    list-style: none; 
+    padding: 1rem 1rem 1rem 1rem;
+    position: relative;
+  }
+li:before {
+    content: " / ";
+    position: absolute;
+    right: 0;
+}
+li:last-child:before { 
```

Comment:
see if you can use the `not` code to combine these. How you wrote yours is similar to mine, but I've seen a few students using `not` and it's rad